src/stylesheets/base/_table.scss

```
@@ -0,0 +1,43 @@
+.table {
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  width: 100%;
+  text-align: left;
+  color: map-get($color-white, 'secondary');
+
+  &__head {
+    font-size: 0.75rem;
+    line-height: 1rem;
+    color: map-get($color-white, 'secondary');
+    background-color: map-get($color-black, 'secondary');
+    text-transform: uppercase;
+  }
+
+  &__head-column {
```

1. A table is composed of cells
2. Instead of using the parent element as a prefix, prefer using the CSS child selector

```suggestion
  &__head > &__cell {
```

---

Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/81/commits/99495156361e47333ffdabf8c07380429a2c5948